uploader/uploader.py

```diff
@@ -230,7 +239,7 @@ def main():
                 if params.mode == 'rsync':
                     exit_code = subprocess.call([
                         "rsync", "-av", "--timeout=300", "--contimeout=300",
-                        "--progress", fname_u, params.url])
+                        "--progress", "--ignore-existing", fname_u, params.url])
```

Comment:
Has anyone tested yet whether this actually works? FOS is a write-only target, so I could imagine that `--ignore-existing` might not work, depending on how it's implemented in rsync. (The client can't get a file listing from a write-only server, so if the check is running on the client, I don't see how this would work. If, on the other hand, the client sends the filename to the server and then gets a reply whether the file exists already, it might work.)

Also, what happens when the file exists on the server already? Does it simply get deleted on the pipeline? (I hope not...)

Reply:
Not required as fixed in uploader.py